infra/helper.py

```diff
@@ -480,12 +475,10 @@ def build_image_impl(project, cache=True, pull=False):
   if not cache:
     build_args.append('--no-cache')
 
-  build_args += ['-t', 'gcr.io/%s/%s' % (image_project, image_name)]
-  if docker_file_path:
-    build_args += [
-        '--file',
-        docker_file_path,
-    ]
+  build_args += [
+      '-t',
+      'gcr.io/%s/%s' % (image_project, image_name), '--file', dockerfile_path
```

Comment:
nit: f-style string.

Reply:
I think we try to support older versions of python in helper.py which means I can't use f-strings.